lib/install/management/create.go

```diff
@@ -18,10 +18,12 @@ import (
 	"context"
 	"fmt"
 	"path"
-	"sync"
+	"path/filepath"
 	"time"
 
 	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/lib/progresslog"
+
```

Comment:
Is there a reason for the extra new line here? Looks like this is all from the same pathing for the most part.

Reply:
Unless `vmware/govmomi` counts as third-party. I've seen govmomi pkgs included in the same group as `vmware/vic` pkgs everywhere in our repo though.

Reply:
No, there should not be a new line. It seems like an autoimport bug.